src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html

```diff
@@ -5,7 +5,7 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
 
 		<title>Datenschutzerklärung &bull; Corona-Warn-App</title>
-		<link rel="stylesheet" href="../style.css">
+		<script type="text/javascript" src="/ruxitagentjs_ICA27SVfqrux_10221210805073954.js" data-dtconfig="app=ea7c4b59f27d43eb|rcdec=1209600000|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|srmcrv=10|cuc=cuve8t91|mel=100000|dpvc=1|ssv=4|lastModification=1628617564535|dtVersion=10221210805073954|srmcrl=1|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10221210805073954.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_-894154638|rpid=1141919234|domain=net.sap"></script><link rel="stylesheet" href="../style.css">
```

Comment:
where does this url come from and why it is needed?

Reply:
it's useless, I have removed it.